xdotool/functions

```diff
@@ -55,14 +57,21 @@ function zsh-notify() {
 
     zstyle -s ':notify:' "$type"-sound notification_sound
     zstyle -s ':notify:' "$type"-icon icon
+    zstyle -s ':notify:' app-name app_name
+    zstyle -s ':notify:' expire-time expire_time
 
     title=$(notification-title "$type" time_elapsed "$time_elapsed")
 
     if [[ -n "$icon" ]]; then
         icon_option="-i $icon"
     fi
 
-    notify-send ${=icon_option} "$title" "$message"
+    if [[ "$app_name" -eq "AUTO" ]]; then
```

Comment:
i don't think we need a special value to have an app_name: if you set the default value as an empty string in `notify.plugin.zsh` (as for example `success-sound`), then here you can check if `app_name` is an empty string, and act accordingly.